clowdapp.yaml

```diff
@@ -423,6 +423,14 @@ parameters:
   value: 100m
 - name: CPU_REQUEST_FLOO
   value: 50m
+- name: MEMORY_LIMIT_IMPORT_SSG
+  value: 2000Mi
```

Comment:
It doesn't help, also not needed. Let's just go back to the default.

Reply:
okay ! left it back to 500Mi to 1000Mi